src/Traits/HasPermissions.php

```diff
@@ -343,9 +343,8 @@ public function getAllPermissions(): Collection
      * Returns permissions ids as array keys
      *
      * @param  string|int|array|\Spatie\Permission\Contracts\Permission|\Illuminate\Support\Collection  $permissions
-     * @return array
      */
-    public function collectPermissions(...$permissions)
+    private function collectPermissions(...$permissions): array
```

Comment:
Note: in #2182 when this `collectPermissions()` method was created, it was created by @angeljqv as `public`, but probably should have been `private`.
It was released in `v5.5.7`.

Since it's a breaking change, albeit highly unlikely to affect anyone, we won't backport this to `v5`.

Reply:
Ok, i did test it, it works correctly for me, seems the right way